drivers/bridge/bridge.go

```diff
@@ -430,7 +431,13 @@ func (d *driver) Config(option map[string]interface{}) error {
 	}
 
 	if config.EnableIPForwarding {
-		return setupIPForwarding(config)
+		if err := setupIPForwarding(config); err != nil {
+			return err
+		}
+	}
+
+	if err := iptables.RemoveExistingChain(DockerChain, iptables.Nat); err != nil {
```

Comment:
Also shouldn't this be done in `Init` rather than in `Config`? `Init` seems to be the right place.